src/main/java/com/mercadopago/client/order/OrderRefundPaymentRequest.java

```diff
@@ -0,0 +1,23 @@
+package com.mercadopago.client.order;
+
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Builder
+@Getter
+@Setter
+/** OrderRefundRequest class. */
+public class OrderRefundPaymentRequest {
```

Comment:
ajustar ordem do comentário e remover setter
```suggestion
/** OrderRefundRequest class. */
@Builder
@Getter
public class OrderRefundPaymentRequest {
```